.github/workflows/build-container.yml

```
@@ -0,0 +1,36 @@
+name: Build container
+
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Checkout the codebase
+        uses: actions/checkout@v2
+      - name: Login to the Container registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ghcr.io/ndom91/briefkasten
```

Small nit: I'd like to also have the version tags and a commit sha tag. If I understood the docs correctly, when using the `type=semver,pattern=...` tag, it will also auto apply the `latest` tag. So you can drop the `flavor: latest=true` part.
```suggestion
          images: ghcr.io/ndom91/briefkasten
          tags: |
            type=semver,pattern={{version}}
            type=sha
```

---

Good hint, I will fix that :)

Would you prefer to push the image to duckerhub? This can also be done in this github actions file.
So there are three options here:

1. Only push to ghcr.io on commit
2. Push to dockerhub and ghcr.io
3. Only push to dockerhub

What would you prefer? Just let me know and I adjust the code